app/storage/encrypted_questionnaire_storage.py

```diff
@@ -22,21 +23,18 @@ def add_or_update(self, data, version):
         questionnaire_state = self._find_questionnaire_state()
         if questionnaire_state:
             logger.debug('updating questionnaire data', user_id=self._user_id)
-            questionnaire_state.state = encrypted_data_json
+            questionnaire_state.state_data = encrypted_data_json
```

Comment:
the repetition of state seems a bit odd? the fact it is state is implied by the object name?
so maybe just `.data`?

Reply:
`data` is also a reserved keyword in dynamo

Reply:
🤦‍♂️